packages/react-components/react-tree/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -189,16 +214,31 @@ export function useTreeItem_unstable(props: TreeItemProps, ref: React.Ref<HTMLDi
       role: 'treeitem',
       'aria-level': level,
       [dataTreeItemValueAttrName]: value,
+      'aria-selected': selectionMode === 'none' ? undefined : checked,
```

Comment:
here we could have `aria-checked` for multi-select and `aria-selected` for single

Reply:
Great idea. I see this is the actual recommendation on ARIA spec! https://www.w3.org/WAI/ARIA/apg/patterns/treeview/